demo/basic/constraints/all_ns_must_have_gatekeeper_dryrun.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
Are we demoing this? It may be worth making sure this feature is surfaced.

Reply:
will add some docs for this after code review is done